### DISCUSSION TITLE
Unable to run C in vs code

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/vscode-cpptools#7245] [Q&A]
Hello Guys
I have an error while I tried to run a C code after configure my setting
`{
    "code-runner.runInTerminal": true,
    "C_Cpp.errorSquiggles": "Disabled"
}`

Then here are the error I get when click on Run code

`PS E:\C C++ basics> cd "e:\C C++ basics\" && gcc shit.c -o shit && "e:\C C++ basics\"shit
The token '&&' is not a valid statement separator in this version.
At line:1 char:25
+ cd "e:\C C++ basics\" && <<<<  gcc shit.c -o shit && "e:\C C++ basics\"shit
    + CategoryInfo          : ParserError: (&&:String) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine`

Then I need helps from you 
Thanks

**Comment by sean-mcmanus** [comment]
I think that error is coming from PowerShell. The "&&" is not valid with the latest version PowerShell. If the CodeRunner extension is responsible you should file an issue there. Maybe you could try using a different shell, such as cmd.
